src/framework/preferences/js/Builder.js

```diff
@@ -196,12 +196,22 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
     };
 
     /*
-     * A one-stop-shop function to build and instantiate a prefsEditor from a schema.
+     * A one-stop-shop component to build and instantiate a prefsEditor from a schema.
      */
-    fluid.prefs.create = function (container, options) {
-        options = options || {};
-        var builder = fluid.prefs.builder(options.build);
-        return fluid.invokeGlobalFunction(builder.options.assembledPrefsEditorGrade, [container, options.prefsEditor]);
-    };
+    fluid.defaults("fluid.prefs.create", {
+        gradeNames: ["fluid.viewComponent"],
+        builder: {}, // options provided by users
+        prefsEditor: {}, // options provided by users
+        members: {
+            constructedGrades: "@expand:fluid.prefs.builder({that}.options.build)"
+        },
+        components: {
+            prefsEditor: {
+                type: "{fluid.prefs.create}.constructedGrades.options.assembledPrefsEditorGrade",
```

Comment:
Our documentation of how to use ```fluid.prefs.create``` to construct preferences editor:
http://docs.fluidproject.org/infusion/development/PreferencesEditor.html

```fluid.prefs.create``` is primarily used by users to provide options that controls the assembling of the preferences editor. For its returned grade, users can also find some builder information such as grade names of assembled prefsEditor or uiEnhancer. But I doubt if users would care about those things. :)